cypress/integration-qe/networkValidation.spec.js

```diff
@@ -0,0 +1,160 @@
+      expect($elem).to.be.disabled;
+    });
+  });
+
+  it('set invalid DNS: empty DNS', () => {
+    cy.get('#form-input-baseDnsDomain-field').clear();
+    cy.get('#form-input-baseDnsDomain-field-helper').contains('The value is required.');
+    cy.get('button[name="save"]', { timeout: VALIDATE_CHANGES_TIMEOUT }).should(($elem) => {
+      expect($elem).to.be.disabled;
+    });
+  });
+
+  it('set invalid DNS: single word', () => {
+    cy.get('#form-input-baseDnsDomain-field').clear().type('invalidDns');
+    cy.get('#form-input-baseDnsDomain-field-helper').contains(
+      'is not valid DNS name. Example: basedomain.example.com',
+    );
+    cy.get('button[name="save"]', { timeout: VALIDATE_CHANGES_TIMEOUT }).should(($elem) => {
+      expect($elem).to.be.disabled;
+    });
+  });
+
+  it('check that correct DNS saved', () => {
+    openCluster(CLUSTER_NAME);
+    cy.get('#form-input-baseDnsDomain-field').should('have.value', DNS_DOMAIN_NAME);
+    cy.get('#form-input-baseDnsDomain-field-helper').contains('The base domain of the cluster.');
+  });
+});
+
+describe('Cluster API VIP Validation', () => {
+  it('open cluster details', () => {
+    openCluster(CLUSTER_NAME);
+  });
+
+  it('get master IP', () => {
+    cy.get('#expandable-toggle0').click();
+    cy.get(nicsTable(1, 'IPv4 address')).then(($elem) => {
+      MASTER_IP = $elem.text().split('/')[0];
+    });
+  });
+
+  it('set invalid IP: IP outside the CIDR', () => {
+    openCluster(CLUSTER_NAME);
+    cy.get('#form-input-apiVip-field').clear().type(VIP_OUT_CIDR);
+    cy.get('#form-input-apiVip-field-helper').contains('IP Address is outside of selected subnet');
+  });
+
+  it('set invalid IP: same IP as master-0-0', () => {
+    cy.get('#form-input-apiVip-field').clear().type(MASTER_IP);
+    saveClusterDetails();
+    cy.get('.pf-c-alert').contains('is already in use in cidr');
+    cy.get('.pf-c-alert__action').click();
+  });
+
+  it('save is not allowed due to empty IP', () => {
+    cy.get('#form-input-apiVip-field').clear();
+    cy.get('button[name="save"]', { timeout: VALIDATE_CHANGES_TIMEOUT }).should(($elem) => {
+      expect($elem).to.be.disabled;
+    });
+    cy.get('#form-input-apiVip-field-helper').contains('The value is required.');
+  });
+
+  it('check that correct VIP saved', () => {
+    cy.get('#form-input-apiVip-field').type(API_VIP);
+    cy.get('button[name="save"]', { timeout: VALIDATE_CHANGES_TIMEOUT }).should(($elem) => {
+      expect($elem).to.be.disabled;
+    });
+    cy.get('#form-input-apiVip-field-helper').contains(
+      'Virtual IP used to reach the OpenShift cluster API.',
+    );
+  });
+});
+
+describe('Cluster Ingress IP Validation', () => {
+  it('open cluster details', () => {
+    openCluster(CLUSTER_NAME);
+  });
+
+  it('get master IP', () => {
+    cy.get('#expandable-toggle0').click();
+    cy.get(nicsTable(1, 'IPv4 address')).then(($elem) => {
+      MASTER_IP = $elem.text().split('/')[0];
```

Comment:
This works just by accident, please see https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html .

One way to fix it is to use `cy.get().within()`, please grep other tests for examples. Or using `.as()` - not yet used in our tests, IIRC.